openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -1197,6 +1203,66 @@ message BidRequest {
     extensions 100 to 9999;
   }
 
+  // OpenRTB 2.6: Structured user agent information, which can be used when a client supports
+  // User-Agent Client Hints. If both device.ua and device.sua are present in the bid request,
+  // device.sua should be considered the more accurate representation of the device attributes.
+  // This is because the device.ua may contain a frozen or reduced user agent string
+  // due to deprecation of user agent strings by browsers.
+  message UserAgent {
+    // Each BrandVersion object identifies a browser
+    // or similar software component. Implementers should send brands and
+    // versions derived from the Sec-CH-UA-Full-Version-List header*.
```

Comment:
The asterisk refers to a footnote that is in the specification though not yet replicated here:

> or an equivalent JavaScript accessor from NavigatorUAData interface. This header or accessor are only available for browsers that support User-Agent Client Hints.

Let's please find a way to include it here (my preferred option), or remove the asterisk. I'd either include it above the table, or repeat it each place the asterisk appears.